modules/View.js

```diff
@@ -56,6 +56,70 @@
       if (!options.noActivate) {
         this.activate();
       }
+      this.updateDelegateEvents();
+    },
+
+
+    /**
+    * @param method {function} callback method of event
+    * @param eventName {String} description of event type 
+    * @return modified mehtod with before/after signals sent to Logger
+    * @method trackEvents
+    */
+    trackEvents : function(method, eventName){
+      var self = this;
+      var methodCopy = method;
+      method = _.bind(function(){
+        var UUID = "uuid-"+(new Date()).getTime().toString(16)+Math.floor(1E7*Math.random()).toString(16);
+        Logger.track({
+          UUID : UUID,
+          type : "clickEvent",
+          state: "start",
+          eventName: eventName,
+          time: Date.now(),
+        });
+        methodCopy.call(self);
+        Logger.track({
+          UUID: UUID,
+          time: Date.now(),
+          state: "end",
+        });
+      },this);
+      return method;
+      },
+
+    /**
+    * updates Backbone's delegateEvents to bind signals to Logger to all callback 
+    * functions in the View's list of events
+    * @method updateDelegateEvents
+    */
+    updateDelegateEvents: function(){
+      var backboneDelegateEvents = Backbone.View.prototype.delegateEvents;
+      Backbone.View.prototype.delegateEvents = function(events){
+        var delegateEventSplitter = /^(\S+)\s*(.*)$/;
```

Comment:
@cmchin, I spoke to @mandragorn about this and if we're going to re-write backbone, we at least need to be able to add hooks for application code. If I, as a programmer using torso, want to add "before" and "after" logic on event callbacks on a view, then it would be worth overriding delegate events.